cijoe/workflows/benchmark_latency.yaml

```diff
@@ -0,0 +1,83 @@
+---
+doc: |
+  Environment prep, benchmark run, result plotting, and report gen.
+  =================================================================
+
+  Environment preparation consists of
+
+  * Retrieving and updating repositories for liburing, xnvme, fio, and SPDK
+  * Configure, build and install liburing, xnvme, fio, and SPDK
+  * Generate bdev-configs for all CIJOE "duts"
+
+  The comparison and plots consists of
+
+    * bandwidth in KiB / seconds
+    * latency is mean and in nano-seconds
+    * IOPS are 'raw' / base-unit
+
+  The above in a bar-plot at iodepth=1, and a line-plot as a function of iodepth.
+
+  See Artifacts for the generated plots
+
+steps:
+
+- name: sysinfo
+  uses: linux.sysinfo
+
+- name: repository
+  uses: core.repository_prep
+
+- name: liburing
+  uses: liburing_build
+
+- name: xnvme_build
+  uses: xnvme_build
+
+- name: xnvme_install
+  uses: xnvme_install
+
+- name: ldconfig
+  run: ldconfig
+
+- name: fio
+  uses: fio_prep
+
+- name: fio_latency
+  uses: fio_prep_latency
```

Comment:
What is the difference between these?

Reply:
We should remove fio_prep. its an artifact from the performance bench

Reply:
Are you sure? It looks like fio_prep_latency is not doing anything